src/related-items-and-comparisons/sampleData.js

```diff
@@ -0,0 +1,158 @@
+export const products = [
```

Comment:
Something you might find useful when you get to the "comparisons" part: these items don't have a `.features` field because they're based on the `GET /products` API endpoint, but since you'll probably be pulling this data from `GET /products/:product_id` instead, you can expect it to have `.features` like [this](https://learn-2.galvanize.com/cohorts/2967/blocks/94/content_files/Front%20End%20Capstone/project-atelier-catwalk/products.md#product-information).